src/daemon/configfile.c

```diff
@@ -414,8 +414,10 @@ static int dispatch_block_plugin(oconfig_item_t *ci) {
   for (int i = 0, ret = 0; i < ci->children_num; i++) {
     if (ci->children[i].children == NULL) {
       ret = dispatch_value_plugin(name, ci->children + i);
-      if (ret != 0)
+      if (ret != 0) {
+        ERROR("dispatch for plugin %s returned non-zero code : %i", name, ret);
```

Comment:
I think we need smth more human-readable, like "Plugin `%s` failed to handle option `%s`, return code: %i".

Reply:
done